crowdfunding/crowdfunding/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from .import models
+
+# Register your models here.
+admin.site.register(models.Projects)
+
+#register link
+admin.site.site_url = "/projects"
```

Comment:
It's because I'm lazy and when I'm in "ADMIN" mode and click on the "VIEW SITE" on the nav, it redirects to projects and not the default "error" page, lol

![image](https://user-images.githubusercontent.com/114335060/224532625-fae38318-8c51-47ec-939e-d952b1bfc0f9.png)

![image](https://user-images.githubusercontent.com/114335060/224532709-76c2beb5-52b4-4a90-b6e3-0024e376f963.png)

Reply:
Oh this makes so much sense!

Nice!